watertap/unit_models/tests/test_osmotically_assisted_reverse_osmosis_1D.py

```diff
@@ -474,7 +489,258 @@ def test_solution(self, RO_frame):
             m.fs.unit.permeate_side.deltaP_stage[0]
         )
 
-    @pytest.mark.requires_idaes_solver
+    # NOTE Begin SKK tests
```

Comment:
Similarly as 0D model, I think maybe we can add a test for more general case with `concentration_polarization_type` and  `mass_transfer_coefficient` set as `cacluated`